draft-ietf-tls-rfc8447bis.md

```diff
@@ -535,6 +535,19 @@ IANA is requested to add a "Comment" column to the following registries:
 This list of registries is all registries that do not already have a
 "Comment" or "Notes" column or that were not orphaned by TLS 1.3.
 
+# Expert Review of Current and Potential IETF and IRTF Documents
+
+The intent of the Specification Required standard for TLS code points
+is to allow for easy registration for code points associated with
+protocols and algorithms that are not being actively developed inside
+IETF or IRTF. When TLS-based technologies are being developed inside
+the IRTF/IETF they should be done in coordination with the TLS WG in
+order to provide appropriate review. For this reason, designated
```

Comment:
When do we expect this coordination to occur?  Would a registration still be reasonable after such coordination but prior to publication as an RFC?  Do we want to consider the case of WGs that produce drafts not intended for publication as an RFC (it would be a bit weird for such things to want to register TLS codepoints, IMO)?

Reply:
I don't think we have to specify this.

However, I think that the answer is "must have TLS WG assent".

Reply:
Well, the current text (mostly a couple lines below) seems to say that if there is a WG/RG draft the experts should push back, full stop.  If we really mean "should push back in the absence of coordination with the TLS WG", we should probably say that.

Reply:
That seems reasonable. Do you have some proposed text?

Reply:
Sean beat me to it.